package/MDAnalysis/analysis/hydrogenbonds/hbond_analysis.py

```diff
@@ -280,9 +280,9 @@ def __init__(self, universe,
             universe topology does not contain bonding information
         d_a_cutoff : float (optional)
             Distance cutoff for hydrogen bonds. This cutoff refers to the D-A distance. [3.0]
```

Comment:
remov default

Reply:
There are more instances of missed defaults in the docs here.

Reply:
Done